linux_os/guide/services/base/group.yml

```diff
@@ -1,6 +1,6 @@
 documentation_complete: true
 
-prodtype: rhel6, rhel7
+prodtype: rhel6, rhel7, fedora
```

Comment:
@jan-cerny can you remove spaces here? For some reason, they have started to appear in the codebase.

Reply:
ok